src/index.html

```diff
@@ -5,9 +5,55 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+
+    <div class="card" data-qa="card">
+
+      <div class="card__image card-top">
+
+        <img
+          src="./images/imac.jpeg"
+          class="img"
```

Comment:
```suggestion
    <div class="card" data-qa="card">
      <img
        src="./images/imac.jpeg"
        class="card__image"
```
Don't add empty lines between parent and child elements.
Also, there is actually no need to create a container for the image. 
You can set all the needed properties to img directly